api-specs/api/resources/in-store.raml

```diff
@@ -173,6 +173,82 @@ uriParameters:
           body:
             application/json:
               example: !include ../examples/cart.example.json
+  /key={key}:
```

Comment:
not sure if I should add it in the /me endpoint  here in the in-store.raml

Reply:
No as the me endpoint doesn't support querying cart by key